core/src/syn/parser/stmt/define.rs

```diff
@@ -210,9 +210,36 @@ impl Parser<'_> {
 						res.roles.push(self.next_token_value()?);
 					}
 				}
-				t!("SESSION") => {
+				t!("DURATION") => {
 					self.pop_peek();
-					res.set_session(Some(self.next_token_value()?));
+					while self.eat(t!("FOR")) {
+						self.eat(t!(","));
```

Comment:
I think we only need the one at end. Currently you could do:
```
DURATION ,FOR X dur,,FOR Y dur,,FOR Z dur,
```

Reply:
You are right! I tried to keep the behavior consistent with the `PERMISSIONS` clause:
https://github.com/surrealdb/surrealdb/blob/main/core/src/syn/parser/stmt/parts.rs#L231-L234
However, on a second look, the first `eat` there is outside of the loop, just to consume the first comma.
Nice catch!